### PR TITLE
Add Azure release v12.1.2

### PR DESCRIPTION
**Pull request by corest**
Towards https://github.com/giantswarm/giantswarm/issues/14069

## TODO
- [x] Test release
- [x] Ping @sig-product for review of release notes.

### CODE REVIEW

**Review by MartaD — APPROVED** [review]
LGTM.

File: azure/v12.1.2/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# :zap: Giant Swarm Release v12.1.2 for Azure :zap:
+
+**Nodes will be rolled during upgrade to this version.**
+
+This patch release prevents an issue with QPS(Queries per Second) limits introduced by DockerHub.
```
Comment (cokiengchiara) [inline azure/v12.1.2/README.md:5]:
```suggestion
This patch release prevents an issue with QPS (Queries per Second) limits introduced by DockerHub.
```

File: azure/v12.1.2/announcement.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+**Giant Swarm release v12.1.2 for Azure is available**. This patch release prevents an issue with QPS(Queries per Second) limits introduced by DockerHub.
```
Comment (cokiengchiara) [inline azure/v12.1.2/announcement.md:1]:
```suggestion
**Giant Swarm release v12.1.2 for Azure is available**. This patch release prevents an issue with QPS (Queries per Second) limits introduced by DockerHub.
```

**Review by cokiengchiara — APPROVED** [review]
Thanks for including the notes from previous upgrade. LGTM

**Pull request merged**